### PR TITLE
Bookstore: styling

### PR DESCRIPTION
**Pull request by Titus-Kiplagat**
- Added missing JSX elements to the project (without adding extra functionality).
- Added CSS styling to match the final [Zeplin](https://zeplin.io/) design.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -6,7 +6,7 @@ import Navigation from './components/Navigation';
 function App() {
   return (
     <Router>
-      <div className="w-11/12 mx-auto h-screen">
+      <div className="mx-auto">
         <Navigation />
         <Routes>
```
Comment (TadesseDev) [inline src/App.js]: - [ ] Deear @Titus-Kiplagat, Your design is really awesome. Yet I belive there are some small changes we can apply to come up with a better layout.

1. The arrangement is a bit off compared with the original design. As you can see in the original design the `user icon` and the `heading/logo text` are exactly straight with the books list/card. Please make sure the book card is aligned with the `user logo` and the `primary text`.  
2. Also the margin for the cards is a bit less, consider adding more margin from left and right.

### Original design 
Screen Shot 2023-09-30 at 5 24 34 PM

### Your design 
Screen Shot 2023-09-30 at 5 18 12 PM
> This change becomes worse as I shift the display size.
Screen Shot 2023-09-30 at 5 29 37 PM

**Other than those minor details, everything else seems perfect ✅ **

**Review by TadesseDev — CHANGES_REQUESTED** [review]
### Hi @Titus-Kiplagat 

### I'm Tadesse Alemayehu your reviewer for this PR. :hand: 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
212094426-b41dd457-c9db-4ab3-bbe9-176dd8dd08b7




## Highlights
 - [x] Good job overall following the template design ✅
 - [x] You have written a clean code ✅
 
## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
- N/A
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/App.js (hunk 1 above)
Comment (Titus-Kiplagat) [reply src/App.js]: Thank you for the suggestion. I completely agree that using padding alone is not the ideal approach for aligning both the `navbar` and the `main section`. This is especially true considering the need for responsiveness across various screen sizes.

**Review by KanzaTahreem — APPROVED** [review]
# Approved 🙌

Hi @Titus-Kiplagat 

Good job implementing the suggested changes 💪 **_Your project is COMPLETE! There is nothing else to say other than... it's time to merge it! 🚀 Congratulations!_**



## Highlights 🎯
There are some excellent points that you have made perfectly. Keep up the great work! 👏

-  ✅ Styling is looking great
-  ✅ Great work using the right GitFlow 
-  ✅ Your README looks good
-  ✅ Linters are working properly

## Optional suggestions ❕
N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👨‍💻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**